scripts/peer-api-check.js

```diff
@@ -32,10 +34,10 @@ if (peerVersion) {
       `Package ${pjson.name} does't have API version pinned in dev dependencies: ${devVersion}`
     );
   }
-  if (peerVersion !== `^${devVersion}`) {
+  if (!semver.satisfies(devVersion, peerVersion)) {
     throw new Error(
       `Package ${pjson.name} depends on peer API version ${peerVersion} ` +
-      `but version ${devVersion} in development`
+      `but version ${devVersion} in development which doesn't satisfy the peer API version`
```

Comment:
```suggestion
      `but version ${devVersion} in development which doesn't satisfy the peer API version ${peerVersion}`
```

Reply:
Thanks for the comment. The error message mentions `peerVersion` in the first part of the message already. Perhaps we could rewrite the whole thing better without duplicating?

Reply:
Sorry I saw this too late. and resolved the conversion just a minute ago.